security/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.devonfw.tools.IDEasy.dev</groupId>
+        <artifactId>ide</artifactId>
+        <version>dev-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.example</groupId>
+    <artifactId>security</artifactId>
+
+    <properties>
+        <java.version>17</java.version>
```

Comment:
why do we override `java.version` here? IMHO we can strip this to avoid extra maintenance effort on Java version updates.

Reply:
It may have been my Intellij not getting the Java language level setting right. So I added this. But I agree, this is not necessary here.